core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -158,7 +158,7 @@ trait Decoder[A] extends Serializable { self =>
    *
    * @param f a function returning either a value or an error message
    */
-  final def trymap[B](f: A => Try[B]): Decoder[B] = new Decoder[B] {
+  final def emapTry[B](f: A => Try[B]): Decoder[B] = new Decoder[B] {
```

Comment:
nice. I had an inconsistency between `trymap` and `instanceTry` with capitalization. This is better.